controllers/admin/AdminModulesController.php

```diff
@@ -1395,10 +1395,29 @@ public function initModal()
     {
         parent::initModal();
 
+        if ($this->checkTranslationSystem(Tools::getValue('configure'))) {
```

Comment:
Maybe, keep trad_link in order to not break other thing but.. in our case, it's useless, juste fix module_name and add 'link'